runtime/nls/shrc/j9shr.nls

```diff
@@ -6644,3 +6644,18 @@ J9NLS_SHRC_SHRINIT_HELPTEXT_FATAL.explanation=NOTAG
 J9NLS_SHRC_SHRINIT_HELPTEXT_FATAL.system_action=
 J9NLS_SHRC_SHRINIT_HELPTEXT_FATAL.user_response=
 # END NON-TRANSLATABLE
+
+J9NLS_SHRC_CM_PRINTSTATS_GC_DISPLAY=%1$d: 0x%2$p GC HINTS KEY: %4$.*3$s Address: 0x%5$p Size: %6$d DATA1: %7$zu DATA2: %8$zu
```

Comment:
This should change to HEAP HINTS

Reply:
The flags are not output.

Reply:
I think the flags are for the internal use only. So I didn't print that out.

Reply:
Not sure what internal use only means. Even if only for debugging purposes it would be nice to have the flags printed.